src/__tests__/auths.test.ts

```diff
@@ -0,0 +1,320 @@
+import mongoose from "mongoose";
+import request from "supertest";
+import dotenv from "dotenv";
+import app from "../app";
+
+
+dotenv.config();
+jest.setTimeout(10000);
+mongoose.set('strictQuery', true);
+const agent = request.agent(app);
+
+beforeEach(async () => {
+  await mongoose.connect(String(process.env.DB_TEST));
+});
+
+/* Closing database connection after each test. */
+afterEach(async () => {
+  await mongoose.connection.close();
+});
+
+describe("Create Company", () => {
+ 
+  it("Should create a new company successfully!", (done) => {
+    const companyInput = {
+      name: "company2",
+      email:"company2@example.com",
+      phoneNumber: "2348080425123",
+      city: "Onitsha",
+      state: "Anambra",
+      password: "mypassword"
+    }
+     
+     agent
+     .post('/api/v1/auths/register/company')
+     .send(companyInput)
+     .expect(200)
+     .end((err, res) => {
+
+       expect(res.status).toEqual(200);
+       return done();
+     })      
```

Comment:
Consider this pattern

```suggestion
     const { status, body } = await agent
       .post('/api/v1/auths/register/company')
       .send(companyInput)

      expect(status).toEqual(200);
      expect(body).toStrictEqual(<expected-body>);
```